examples/bundled-example/main.js

```diff
@@ -55,24 +56,22 @@ const uppy = Uppy({
     closeModalOnClickOutside: false,
     locale: {
       strings: {browse: 'browse'}
-    }
+    },
+    metaFields: [
+      { id: 'license', name: 'License', value: 'Creative Commons', placeholder: 'specify license' },
+      { id: 'caption', name: 'Caption', value: '', placeholder: 'describe what the image is about' }
+    ]
```

Comment:
we should add docs for the `metaFields` option. could do it here or in a followup pr.